app/views/layouts/application.html.erb

```diff
@@ -36,6 +36,7 @@
     )%>);
 
     new ELMO.Views.MissionChangeDropdown({el: '#change_mission select'});
+    new ELMO.Views.RegenerableFieldView();
```

Comment:
That probably makes more sense anyways, since it's a view for the field, not the whole page.